.github/workflows/archive-full-index.yaml

```diff
@@ -0,0 +1,146 @@
+name: Generate the full index archive for Bitnami Charts
+on:
+  schedule:
```

Comment:
Any reason why you can't just execute it on push of the index branch?

on:
  push:
    branches:
    - index

Reply:
Hi @MichelZ , the idea is to run it once per day because usually the truncated index that is downloaded is generated several times per day so we need to generated new full index, at least, once per day in order to be on track.

Reply:
Yes, but then you still "lag" behind the truncated index. Also why would you download the truncated one from CDN and not from the index branch?

Reply:
The point regarding the CDN version of the index is a very good one, @randradas : no real reason to download that file externally when we can just use our repository version of that file, right?

Reply:
right, I'm changing this part to get the file internally from the repo.

Reply:
done!, very good idea :D